### ISSUE TITLE
Should vagrant use dev or production requirements

### ISSUE THREAD

**Issue by vkurup** [caktus/margarita#73]
I tried to set up celery today and when testing on vagrant, celery complained that the memcached module was not set up, which is because we are using dev.txt instead of production.txt:

https://github.com/caktus/margarita/blob/develop/project/venv.sls#L22-L26

Given that we use `deploy.py` settings for vagant, we should probably switch that right?

**Comment by mlavin** [comment]
I don't think we've ever agreed on this. Some people want to use Vagrant as a local dev environment and others use it to test the deployment (staging/production). I can't be both.

**Comment by dpoirier** [comment]
Maybe we need two Vagrant-based environments, one for local development and one for testing deploys?  Maybe env "local" for the first and env "vagrant" for the second?

**Comment by mlavin** [comment]
There is a related problem in `project/repo.sls` in that it assumes that the Vagrant deployment is run from the project repo. We've had more projects where this isn't the case. I don't think this should use the mounted file system if you are trying to replicate a prod-like environment.

**Comment by dpoirier** [comment]
What's the use case for not running the Vagrant deploy from the project repo?  I'm having trouble imagining how that would work.

**Comment by mlavin** [comment]
For cases like Ona, ServiceInfo, or in my case Tracpro where the deployment is managed in a repo separate from the project source.

**Comment by dpoirier** [comment]
Oh I see.  I think I'd still refer to our repo for that as our project repo, which led to my confusion.

**Comment by mlavin** [comment]
Yes I mean specifically that https://github.com/caktus/margarita/blob/develop/project/repo.sls#L26 assumes that the project source tree is mounted in the `/vagrant/` which is not always the case.
